### PR TITLE
Snapshot of APIs as of Apache NetBeans 11.3

### PR DESCRIPTION
**Pull request by ebarboni**
Snapshot of APIs as of Apache NetBeans 11.3 - 6b879cb782eaa4f13a731aff82eada11289a66f7

### CODE REVIEW

File: platform/openide.util.ui/nbproject/org-openide-util-ui.sig [hunk 1]
```diff
@@ -699,6 +699,18 @@ supr java.lang.Object
 hfds LOG,TRANS_LOCK,operatingSystem,pathURIConsistent,transExp,transLoader
 hcls RE
 
+CLSS public abstract org.openide.util.CachedHiDPIIcon
```
Comment (JaroslavTulach) [inline platform/openide.util.ui/nbproject/org-openide-util-ui.sig]: Added by @eirikbakke SVG efforts.

File: platform/openide.util.ui/nbproject/org-openide-util-ui.sig [hunk 2]
```diff
@@ -1711,6 +1723,22 @@ meth public abstract !hasdefault java.lang.String position()
 meth public abstract java.lang.Class<?>[] serviceType()
 meth public abstract java.lang.String path()
 
+CLSS public abstract interface org.openide.util.spi.MutexEventProvider
```
Comment (JaroslavTulach) [inline platform/openide.util.ui/nbproject/org-openide-util-ui.sig]: Interesting. I haven't noticed that during the coding.

File: platform/openide.util.ui/nbproject/org-openide-util-ui.sig [hunk 3]
```diff
@@ -1711,6 +1723,22 @@ meth public abstract !hasdefault java.lang.String position()
...
+meth public abstract org.openide.util.spi.MutexImplementation createMutex()
+
+CLSS public abstract interface org.openide.util.spi.MutexImplementation
```
Comment (JaroslavTulach) [inline platform/openide.util.ui/nbproject/org-openide-util-ui.sig]: New SPI for `Mutex`, I see.

File: platform/openide.util.ui/nbproject/org-openide-util-ui.sig [hunk 4]
```diff
@@ -1711,6 +1723,22 @@ meth public abstract !hasdefault java.lang.String position()
...
+meth public abstract <%0 extends java.lang.Object> {%%0} readAccess(org.openide.util.Mutex$ExceptionAction<{%%0}>) throws org.openide.util.MutexException
+meth public abstract <%0 extends java.lang.Object> {%%0} writeAccess(org.openide.util.Mutex$ExceptionAction<{%%0}>) throws org.openide.util.MutexException
+meth public abstract boolean isReadAccess()
+meth public abstract boolean isWriteAccess()
+meth public abstract void postReadRequest(java.lang.Runnable)
+meth public abstract void postWriteRequest(java.lang.Runnable)
+meth public abstract void readAccess(java.lang.Runnable)
+meth public abstract void writeAccess(java.lang.Runnable)
+
+CLSS public abstract interface org.openide.util.spi.SVGLoader
```
Comment (JaroslavTulach) [inline platform/openide.util.ui/nbproject/org-openide-util-ui.sig]: `SVGLoader` is expected change - related to work by @eirikbakke.

**Comment by ebarboni** [comment]
thks for the review. Merging

**Pull request merged**